client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDriverUri.java

```diff
@@ -33,15 +33,19 @@ public final class TrinoDriverUri
     private static final String JDBC_URL_START = JDBC_URL_PREFIX + "trino:";
 
     private TrinoDriverUri(String uri, Properties driverProperties)
-            throws SQLException
     {
         super(parseDriverUrl(uri), driverProperties);
     }
 
-    public static TrinoDriverUri create(String url, Properties properties)
+    public static TrinoDriverUri createDriverUri(String url, Properties properties)
```

Comment:
Nitpick level over 9000 - `createDriverUri` stutters with the class name.

Reply:
yes but clashes with the parent method that doesn't throw SQLException